### DISCUSSION TITLE
Little improvement to efficiently mode

### DISCUSSION THREAD

**Discussion by bigbruno** [linuxdeepin/developer-center#2961] [Features Request & Ideas | 特性请求 & 头脑风暴]
Using the fashion mode makes it easier to notice when a program is open, as there is a white line down the icon, but effectively mode there is no such identification, only a slightly lighter square in the icon background.

This square icon background when using the "efficient mode", it is very difficult to visualize in different screen types that have lower contrast, I think it would be good for many users who were intensified the difference between open programs and not open in efficiently mode.

**Comment by ssk-wh** [comment]
Hmm, is there any better suggestion?
